cogip/tools/planner/camp.py

```diff
@@ -0,0 +1,29 @@
+from enum import IntEnum
+
+from cogip.utils.singleton import Singleton
+
+
+class Camp(metaclass=Singleton):
+    """
+    Class representing the camp selected before the game starts.
+    """
+    class Colors(IntEnum):
+        blue = 0
+        green = 1
+
+    def __init__(self, color: Colors = Colors.blue):
+        self.color = color
+
+    def adapt_y(self, dist: float) -> float:
```

Comment:
Dont remember, are you sure it is Y and not X ?

Reply:
Yes sûre 😉